packages/wix-ui-tpa/src/components/Dialog/Dialog.tsx

```diff
@@ -131,5 +135,9 @@ export class Dialog extends React.Component<DialogProps> {
         )}
       </TPAComponentsConsumer>
     );
+
+    return appendToBody
```

Comment:
I think we might need to add a check that `document` exists, so we don't break ssr
```suggestion
    return appendToBody && typeof document !== 'undefined'
```

Reply:
Good point 👍